target/linux/ipq807x/image/generic.mk

```diff
@@ -108,6 +108,18 @@ define Device/netgear_wax218
 endef
 TARGET_DEVICES += netgear_wax218
 
+define Device/prpl_haze
+	$(call Device/FitImage)
+	$(call Device/EmmcImage)
+	DEVICE_VENDOR := prpl Foundation
+	DEVICE_MODEL := Haze
+	DEVICE_DTS_CONFIG := config@hk09
+	SOC := ipq8072
+	DEVICE_PACKAGES += ath11k-firmware-qcn9074 ipq-wifi-prpl_haze kmod-ath11k-pci
+	DEFAULT_PACKAGES += ath11k-firmware-qcn9074 ipq-wifi-prpl_haze kmod-ath11k-pci
```

Comment:
Why are you extending the target `DEFAULT_PACKAGES`?

Reply:
To have those packages right in the initramfs, so can test automatically wireless part in the [daily runtime CI testing](https://gitlab.com/ynezz/openwrt-device-runtime-testing/-/pipelines/888312563/builds) without any additional steps. I plan to replace current ipq807x DUT Dynalink WRX36 with Haze as it have more features.

Reply:
Is this working for you?
For me, it does not expand the default package list.
Also, this would mean that all boards would have to carry these and I dont think that is the desired effect.

Reply:
It doesn't work as expected, indeed.